app/src/main/java/org/mozilla/tv/firefox/IntentValidator.kt

```diff
@@ -81,4 +84,15 @@ object IntentValidator {
         }
         return null
     }
+
+    private fun setExperimentOverrides(intent: SafeIntent, context: Context) {
+        intent.extras?.getStringArray(ACTIVE_EXPERIMENTS_KEY)?.let { strArray ->
+            val fretboard = context.serviceLocator.fretboardProvider.fretboard
+            fretboard.clearAllOverrides(context)
```

Comment:
If we receive an Intent while the app is foregrounded, we're going to clear all of our overrides: is that intended? I'd think maybe we want to clear our overrides in `onStop`, or `onCreate`, or something... Why is clearing necessary?

Reply:
I realized this only happens conditionally, when an intent is received and new extras are set, which makes sense: when overrides are received, clear all overrides. Should we document the behavior somewhere for QA?

Reply:
I'll send an email out, and ask if there's another place QA would like that documentation (or put it in the README).